libs/geo/build.gradle

```diff
@@ -21,6 +21,16 @@ apply plugin: 'elasticsearch.build'
 apply plugin: 'nebula.maven-base-publish'
 apply plugin: 'nebula.maven-scm'
 
+archivesBaseName = 'elasticsearch-geo'
```

Comment:
This change should not be needed since we have `project(":libs:geo").name = 'elasticsearch-geo'` in `settings.gradle`. Do you get an error without this ?

Reply:
@atorok when I tried to do that I have got:
```
* Where:
Build file '/home/igor/Projects/Elastic/7.x/elasticsearch/build.gradle' line: 279

* What went wrong:
A problem occurred configuring root project 'elasticsearch'.
> Project :libs:geo not found.
```

Reply:
you have to reference it by the changed name as `:libs:elasticsearch-geo`

Reply:
I get the same thing only with a different name:
```
FAILURE: Build failed with an exception.

* Where:
Settings file '/home/igor/Projects/Elastic/7.x/elasticsearch/settings.gradle' line: 155

* What went wrong:
A problem occurred evaluating settings 'elasticsearch'.
> Project with path ':libs:elasticsearch-geo' could not be found.
```

Reply:
Maybe you forgot to revert the change in settings.gradle ? 
This work for me (I can `./gradlew assemble`): 
```
diff --git a/build.gradle b/build.gradle
index 8f359365794..c611cb7cedb 100644
--- a/build.gradle
+++ b/build.gradle
@@ -212,7 +212,7 @@ allprojects {
     "org.elasticsearch:elasticsearch-core:${version}": ':libs:core',
     "org.elasticsearch:elasticsearch-nio:${version}": ':libs:nio',
     "org.elasticsearch:elasticsearch-x-content:${version}": ':libs:x-content',
-    "org.elasticsearch:elasticsearch-geo:${version}": ':libs:geo',
+    "org.elasticsearch:elasticsearch-geo:${version}": ':libs:elasticsearch-geo',
     "org.elasticsearch:elasticsearch-secure-sm:${version}": ':libs:secure-sm',
     "org.elasticsearch.client:elasticsearch-rest-client:${version}": ':client:rest',
     "org.elasticsearch.client:elasticsearch-rest-client-sniffer:${version}": ':client:sniffer',
diff --git a/libs/geo/build.gradle b/libs/geo/build.gradle
index fdfdc5c7b12..ab3419b93b9 100644
--- a/libs/geo/build.gradle
+++ b/libs/geo/build.gradle
@@ -21,16 +21,6 @@ apply plugin: 'elasticsearch.build'
 apply plugin: 'nebula.maven-base-publish'
 apply plugin: 'nebula.maven-scm'
 
-archivesBaseName = 'elasticsearch-geo'
-
-publishing {
-    publications {
-        nebula {
-            artifactId = archivesBaseName
-        }
-    }
-}
-
 dependencies {
     if (isEclipse == false || project.path == ":libs:geo-tests") {
         testCompile("org.elasticsearch.test:framework:${version}") {
diff --git a/settings.gradle b/settings.gradle
index a9f08e53cb0..c098dc2c723 100644
--- a/settings.gradle
+++ b/settings.gradle
@@ -152,3 +152,4 @@ if (extraProjects.exists()) {
 
 project(":libs:cli").name = 'elasticsearch-cli'
 project(":libs:ssl-config").name = 'elasticsearch-ssl-config'
+project(":libs:geo").name = 'elasticsearch-geo'

```

Reply:
@atorok I could swear that I did exactly that and server couldn't pick up geo as a dependency :-)  except... I just repeated that and it works. So I must have missed something. Thanks!